mysql-test/suite/binlog_encryption/t/binlog_encryption_without_keyring-master.opt

```diff
@@ -0,0 +1 @@
+--skip-core-file --log-error=$MYSQLTEST_VARDIR/tmp/binlog_encryption_without_keyring.err
```

Comment:
This file needs to be deleted in order for MTR --repeat to work and to do that portably the testcase needs to start with the default error log, restart with this one, work, restart with the default one, delete.

Reply:
Done